src/modules/ProductCreator.rb

```diff
@@ -3669,6 +3659,30 @@ def CopyPackages
           )
           Builtins.y2milestone("Compressed metadata: %1", compressed_meta)
 
+          # TODO FIXME get datadir from the source
+
+          # remove the translation files, they would get out of sync with the main data
+          # and confuse libzypp (bsc#1165247)
+          ret = ret &&
+            Exec(
+              Builtins.sformat(
+                "rm %1/%2/%3/setup/descr/packages.*",
+                String.Quote(basedir),
```

Comment:
This will not work when the `basedir` contains a space. There were quite some bug reports about this in the past, it seems users really use spaces in directory names... :wink: 

[String.Quote](https://github.com/yast/yast-yast2/blob/66bc64d08c6ff017b88e3cff63fd02d78bbcf7f1/library/types/src/modules/String.rb#L57) is designed to be used inside **single quotes**, so this quoting will not work properly.

As we cannot use single quotes here I'd use `Shellwords` here:

```ruby
basedir.shellescape,
subdir.shellescape,
datadir.shellescape
```

(+ add `require "shellwords"` ad the top).